indico/modules/search/client/js/components/SearchBar.jsx

```diff
@@ -0,0 +1,33 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import {Form} from 'semantic-ui-react';
+import {Translate} from 'indico/react/i18n';
+
+export default function SearchBar({onSearch, searchTerm}) {
+  const [value, setValue] = useState(searchTerm);
+
+  const handleChange = event => {
```

Comment:
```suggestion
  const handleChange = (event: {target: {value}}) => {
```

Reply:
(personal preference)

Reply:
i'm not a huge fan of destructuring for event handlers (just personal preference as well)